### PR TITLE
[TinyTodo] Update the instruction to use Python virtual environment

### PR DESCRIPTION
**Pull request by shaobo-he-aws**
*Issue #, if available:*
#139 

*Description of changes:*

### CODE REVIEW

File: tinytodo/README.md [hunk 1]
```diff
@@ -14,9 +14,11 @@ You need Python3 and Rust. Rust can be installed via [rustup](https://rustup.rs)
 
 This example expects that the [`cedar`](https://github.com/cedar-policy/cedar) repository is cloned into the toplevel (`../cedar-examples`) directory. You can instruct Cargo to use your local version of `cedar-policy` by adding `path = "../cedar/cedar-policy"` to Cargo.toml.
 
-Install the needed python packages, and build the server as follows.
+Install the needed python packages in a virtual environment, and build the server as follows.
 
 ```shell
+python3 -m venv /tmp/tinytodo
+source /tmp/tinytodo/bin/activate
```
Comment (cdisselkoen) [inline tinytodo/README.md:21]: is it a good idea / best practice to put the virtual environment in `tmp`?  I think I've previously just put the virtual environment in the current directory, but not sure if that's good or not

**Review by cdisselkoen — APPROVED** [review]

File: tinytodo/README.md (hunk 1 above)
Comment (john-h-kastner-aws) [reply tinytodo/README.md:21]: From my limited python experience I would expect the virtual env to be in the project directory

Comment (cdisselkoen) [reply tinytodo/README.md]: That seems reasonable.  In fact, now that you say that, I feel like I've typed `python3 -m venv venv` before

Comment (john-h-kastner-aws) [reply tinytodo/README.md]: yeah, that's what I assumed craig meant initially

**Pull request merged**